samples/nodejs-compat-streams-split2/README.md

```diff
@@ -0,0 +1,28 @@
+# Node.js Compat Example
+
+To run the example on http://localhost:8080
+
+```sh
+$ ./workerd serve config.capnp
```

Comment:
For the sake of the lowest possible barrier to users, it might be good to make the paths in the README.md align with build outputs / samples that someone might have locally, e.g.

```
$ bazel-bin/src/workerd/server/workerd serve samples/nodejs-compat-streams-split2/config.capnp
```